tests/unit/gapic/recommendationengine_v1beta1/test_prediction_api_key_registry.py

```diff
@@ -18,6 +18,7 @@
 
 import grpc
 from grpc.experimental import aio
+import json
```

Comment:
Adding imports seems pointless (and lint-breaking) without also adding code which uses them.